Baskets.UnitTests/HandlersTests/BasketItemHandlersTests.cs

```diff
@@ -0,0 +1,300 @@
+using Baskets.BusinessLogic.CQRS.Comands.BasketItemComands.CreateBasketItemComand;
+using Baskets.BusinessLogic.CQRS.Comands.BasketItemComands.DeleteBasketItemComand;
+using Baskets.BusinessLogic.CQRS.Comands.BasketItemComands.UpdateBasketItemComand;
+using Baskets.BusinessLogic.CQRS.Queries.BasketItemQueries.GetBasketItemQuery;
+using Baskets.BusinessLogic.CQRS.Queries.BasketItemQueries.GetBasketItemsQuery;
+
+namespace Baskets.UnitTests.HandlersTests
+{
+    public class BasketItemHandlersTests
+    {
+        private readonly UnitOfWorkMock _unitOfWorkMock = new();
+
+        private readonly Mapper _mapper = new(
+            new MapperConfiguration(mc => mc.AddProfiles(
+            [
+                new BasketItemProfile(),
+                new ItemProfile(),
+                new UserBasketProfile()
+            ])));
+
+        public BasketItemHandlersTests()
+        {
+            DataGenerator.InitBogusData();
+
+            _unitOfWorkMock.GetBasketByCondition(DataGenerator.UserBaskets[0]);
+            _unitOfWorkMock.GetAllBasketItems(DataGenerator.BasketItems);
+        }
+
+        [Fact]
+        public async Task GetBasketItems_ValidParameters_ReturnsList()
+        {
+            //Arrange
+            var getBasketItemsQuery = new GetBasketItemsQuery(It.IsAny<string>());
+            var handler = new GetBasketItemsHandler(_unitOfWorkMock.Object, _mapper);
+
+            //Act
+            var response = await handler.Handle(getBasketItemsQuery, cancellationToken: default);
+
+            //Assert
+            response.Should().HaveCount(DataGenerator.BasketItems.Count);
+        }
+
+        [Fact]
+        public async Task GetBasketItems_InvalidBasketId_ThrowsNotFoundException()
+        {
```

Comment:
'async' must be added to the first part of the name if asynchronous functions are being tested inside the test